src/__tests__/media-query-test.js

```diff
@@ -360,45 +364,35 @@ describe('Media query tests', () => {
       <span style={{'@media (min-width: 10px)': {background: 'green'}}} />
     ));
 
+    let error;
+
     class ErrorBoundary extends React.Component {
-      componentDidCatch() {}
+      componentDidCatch(e) {
+        error = e;
+      }
+
       render() {
         return this.props.children;
       }
     }
 
-    const TestComponent = Radium(() => (
+    const TestComponent = () => (
       <ErrorBoundary>
         <ChildComponent />
       </ErrorBoundary>
-    ));
+    );
 
-    // React 16 - need to handle exceptions globally.
-    // In DEV (aka our tests), need to silence global error handlers and such.
-    // https://github.com/facebook/react/issues/10474#issuecomment-322909303
-    window.onerror = sinon.spy();
+    // // React 16 - need to handle exceptions globally.
+    // // In DEV (aka our tests), need to silence global error handlers and such.
+    // // https://github.com/facebook/react/issues/10474#issuecomment-322909303
```

Comment:
Small nit: double comment here `// //`

Reply:
Yep, had commented this and some associated lines out for a moment while trying something, and forgot to un-comment the comment 😂 after putting things back